x-pack/metricbeat/module/enterprisesearch/stats/data.go

```diff
@@ -0,0 +1,108 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package stats
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/elastic/beats/v7/libbeat/common"
+	s "github.com/elastic/beats/v7/libbeat/common/schema"
+	c "github.com/elastic/beats/v7/libbeat/common/schema/mapstriface"
+)
+
+var (
+	connectorsPoolSchema = s.Schema{
+		"queue_depth":     c.Int("queue_depth"),
+		"size":            c.Int("size"),
+		"busy":            c.Int("busy"),
+		"idle":            c.Int("idle"),
+		"total_scheduled": c.Int("total_scheduled"),
+		"total_completed": c.Int("total_scheduled"),
+	}
+
+	schema = s.Schema{
+		"http": c.Dict("http", s.Schema{
+			"connections": c.Dict("connections", s.Schema{
+				"current": c.Int("current"),
+				"max":     c.Int("max"),
+				"total":   c.Int("total"),
+			}),
+
+			"request_duration": c.Dict("request_duration_ms", s.Schema{
+				"max.msec":     c.Int("max"),
+				"mean.msec":    c.Int("mean"),
```

Comment:
I'm pretty sure that we cannot / shouldn't use dots here. I was trying to find an example in our repo that uses it but we don't have any. The problem is I don't remember the reason (maybe I never knew it).

This and all the other fields should be like this example:
```go
"max" : s.Object {
  "ms": c.Int("max"),
}
```